autoload/easy_align.vim

```diff
@@ -89,9 +89,32 @@ function! s:floor2(v)
   return a:v % 2 == 0 ? a:v : a:v - 1
 endfunction
 
+function! s:get_highlight_group_name(line, col)
+  let hl = synIDattr(synID(a:line, a:col, 0), 'name')
+
+  if hl == '' && has('nvim-0.9.0')
+	let insp = luaeval( 'vim.inspect_pos and vim.inspect_pos( nil, ' .. (a:line-1) .. ', ' .. (a:col-1) .. ' ) or { treesitter = {} }' )
```

Comment:
We use 2-space indentation. Here's a tab. And no space around the function parameters.
```suggestion
    let insp = luaeval('vim.inspect_pos and vim.inspect_pos( nil, ' .. (a:line-1) .. ', ' .. (a:col-1) .. ' ) or { treesitter = {} }')
```